README.md

```diff
@@ -0,0 +1,29 @@
+# High Level Controller
+
+## Overview
+
+The control-high repository consists of the High-Level Controller and the MHIST submodule of the SOZO Robotics Project.
+
+The High-Level Controller (HLC) receives the area to cover from the Swarm Algorithm (tbd, currently mocked), gets updates about the local map and the current position in the map from [SLAM](https://github.com/codeuniversity/slam) and feedback about the (sucessful) leg movement from the [gait controller](https://github.com/codeuniversity/control-gait).
+The Planning Alogorithm creates the high level actions (hla) to ensure a sucessful exploration of the area the robot should cover. You can test the algorithms with the pytests defined in /test. These hla ("move forward" or "turn by 90 degrees") are forwarded to the gait controller.
```

Comment:
I'm not sure what `The Planning Alogorithm creates the high level actions (hla) to ensure a sucessful exploration of the area the robot should cover. ` means. How do the high level actions help?

`+` (personal preference) `hla ` -> `HLA` so that it's easier to see that this is an abbreviation 

(could be a personal thing): To me `forwarded` sounds like the actions were received by the controller and are now forwarded; maybe `sent`  is a bit more exact?